integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshUtil.java

```diff
@@ -0,0 +1,99 @@
+package com.sequenceiq.it.cloudbreak;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.schmizz.sshj.SSHClient;
+import net.schmizz.sshj.common.IOUtils;
+import net.schmizz.sshj.connection.channel.direct.Session;
+import net.schmizz.sshj.transport.verification.PromiscuousVerifier;
+
+
+public class SshUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(com.sequenceiq.it.cloudbreak.SshUtil.class);
+
+    public Boolean ssh(String host, String defaultPrivateKeyFile, String sshCommand, String checkType, String value) {
```

Comment:
This method and class doesn't contain any instance specific thing, so should be a static method with a private constructor. XyzUtil also refers that behavior